OpenRA.Mods.Common/Activities/Air/FlyIdle.cs

```diff
@@ -41,6 +53,17 @@ public override bool Tick(Actor self)
 			if (aircraft.ForceLanding || IsCanceling)
 				return true;
 
+			currentFacing = self.TraitOrDefault<IFacing>().Facing;
```

Comment:
Again I think this should be cached. Looking at the lines above it may not be possible. If so, ignore my comment.